calico/felix/endpoint.py

```diff
@@ -453,6 +465,12 @@ def _apply_endpoint_update(self):
                     # table.
                     _log.debug("IP addresses changed, need to update routing")
                     self._device_in_sync = False
+                for key in "ipv4_nat", "ipv6_nat":
```

Comment:
Please can you add a UT for this.  I think we have good coverage in this method so I'd like ot keep it up.